packages/expo-location/src/GeolocationPolyfill.ts

```diff
@@ -11,12 +11,22 @@ type GeolocationOptions = {
   enableHighAccuracy?: boolean;
 };
 
+declare const global: any;
```

Comment:
```suggestion
declare const global: { window?: { navigator?: { geolocation?: { getCurrentPosition?: any } } } };
```
Kick the any down the road a little bit. Or ignore, doesn't matter that much.